target/linux/ramips/image/mt7621.mk

```diff
@@ -544,6 +544,23 @@ define Device/linksys_ea7500-v2
 endef
 TARGET_DEVICES += linksys_ea7500-v2
 
+define Device/linksys_ea7300
```

Comment:
This whole block is mostly a repetition of the device before. Please create a shared definition as e.g. with Device/netgear_sercomm_nand and then inherit the devices from it.
Despite, ea7300 should come _before_ ea7500.